subprojects/platform-play/src/main/java/org/gradle/play/internal/twirl/TwirlCompilerFactory.java

```diff
@@ -20,11 +20,24 @@
 import org.gradle.play.platform.PlayPlatform;
 
 public class TwirlCompilerFactory {
+
     public static TwirlCompiler create(PlayPlatform playPlatform) {
-        return new TwirlCompiler(createAdapter(playPlatform));
+        switch (PlayMajorVersion.forPlatform(playPlatform)) {
+            case PLAY_2_6_X:
+                // TODO: create a TwirlJavaCompiler for Twirl 1.3.x that uses the Java compiler interface
+                // Rename TwirlCompiler to TwirlScalaCompiler
+                // Waiting for https://github.com/playframework/twirl/pull/136
```

Comment:
https://github.com/playframework/twirl/pull/136 was already merged and released. Maybe this can be updated to use `play.japi.twirl.compiler.TwirlCompiler`.